addOns/websocket/src/main/javahelp/org/zaproxy/zap/extension/websocket/resources/help/contents/pscanrules.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+    <HEAD>
+        <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+        <TITLE>
+            WebSocket Passive Scan Rules
+        </TITLE>
+    </HEAD>
+    <BODY BGCOLOR="#ffffff">
+        <H1>WebSocket Passive Scan Rules</H1>
+        <h2 id="scripts">Scripts</h2>
+        Scripts which are included by default in the add-on and they implement the following WebSocket passive scan rules:
```

Comment:
Was there meant to be a list here?

Reply:
Yes, but I think it will be added in the respective PRs (e.g. #2088).

Reply:
Okay

Reply:
Yeap, that's the plan